bokehjs/src/coffee/models/plots/plot_canvas.coffee

```diff
@@ -87,10 +87,6 @@ export class PlotCanvasView extends DOMView
 
     # compat, to be removed
     @frame = @model.frame
-    @x_range = @frame.x_ranges['default']
-    @y_range = @frame.y_ranges['default']
-    @xscale = @frame.xscales['default']
-    @yscale = @frame.yscales['default']
```

Comment:
I couldn't find these in use anywhere, in BokehJS or in any examples. They were marked "to be removed" years ago so now seems like a good time